From 63b99a34f465def97450946cecbed16e9e997779 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@stroblindustries.com>
Date: Sun, 3 Mar 2019 18:48:05 +0200
Subject: [PATCH 1/1] Revert "Gtk-3.2x&4.0: [Budgie-desktop] Stop using
 action-area button styling in IconTaskListApplet popovers"

This reverts commit 123238890da033e6438bb17c663775b38fe99e68.
---
 src/gtk/sass/common/_3rd-party.scss | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/src/gtk/sass/common/_3rd-party.scss b/src/gtk/sass/common/_3rd-party.scss
index 10b49de..77e3698 100644
--- a/src/gtk/sass/common/_3rd-party.scss
+++ b/src/gtk/sass/common/_3rd-party.scss
@@ -392,15 +392,9 @@ stack.icon-popover-stack {
   }
 
   grid.horizontal { // bottom button container
-    > button.flat.image-button { // switch to outlined button array
+    > button.flat.image-button {
+      @extend %action-area-button;
       min-height: $requested_height;
-      margin: 4px 3px;
-      @include radius($r_2);
-      border: 1px solid transparent;
-      @include button(normal);
-      &:hover { @include button(hover); }
-      &:active { @include button(active); }
-      &:disabled { @include button(disabled); }
     }
   }
 }
-- 
2.21.0

